seaborn/utils.py

```diff
@@ -21,6 +21,8 @@
 __all__ = ["desaturate", "saturate", "set_hls_values", "move_legend",
            "despine", "get_dataset_names", "get_data_home", "load_dataset"]
 
+DATASET_NAMES_URL = "https://raw.githubusercontent.com/mwaskom/seaborn-data/master/dataset_names.txt"
```

Comment:
Suggested refactor that will also fix the lint errors:

```suggestion
DATASET_SOURCE = "https://raw.githubusercontent.com/mwaskom/seaborn-data/master"
DATASET_NAMES_URL = f"{DATASET_SOURCE}/dataset_names.txt"
```

And then in `load_dataset` change use `DATASET_SOURCE` to build the url for the csv file.